src/styles/main.scss

```diff
@@ -20,88 +20,78 @@ $dark-color: black;
 $light-grey: lightgrey;
 $background-color: #f9f9f9;
 
-.view-wrapper {
+.react-digraph {
   height: 100%;
   width: 100%;
   margin: 0;
-  display: flex;
   box-shadow: none;
   background: $background-color;
-  transition: opacity 0.167s;
-  opacity: 1;
   outline: none;
   user-select: none;
 
-  > .graph {
-    align-content: stretch;
-    flex: 1;
+  .graph {
```

Comment:
Avoid using non-direct child selectors. These are sometimes useful for selecting any element of a type within a tree, but we would like react-digraph to be very specific about which elements it selects since we don't know which classes a user might define. For instance, if a node has an element with the class `.graph` then this style would suddenly apply to their element as well as the graph component.

Please revert back to using `> .graph {`